recipes/catalystcoop.pudl/LICENSE.txt

```diff
@@ -1,21 +0,0 @@
-The MIT License
```

Comment:
You shouldn't have to delete the files for `catalystcoop.pudl` since the CI during merge will take care of this.